solr_to_es/__main__.py

```diff
@@ -69,15 +69,15 @@ def parse_args():
 
 
 def main():
-    try:
+    try:               
         args = parse_args()
         es_conn = Elasticsearch(hosts=args['elasticsearch_url'], timeout=args['es_timeout'])
 
         # Split the solr_url into the root and the request handler
         solr_conn = pysolr.Solr(args['solr_url'].rsplit('/', 1)[0], search_handler=args['solr_url'].rsplit('/', 1)[-1])
```

Comment:
Can we remove this?

Reply:
thanks, yes pushed another commit to avoid this change